lib/suma/fixtures/offerings.rb

```diff
@@ -46,4 +46,9 @@ module Suma::Fixtures::Offerings
   decorator :with_constraints, presave: true do |*constraints|
     constraints.each { |c| self.add_eligibility_constraint(c) }
   end
+
+  decorator :max_cumulative_and_per_member do |max_cumulative, max_per_member|
```

Comment:
This is a confusing decorator. There's no dependency between these two fields. Remove it.